deployment/docker-compose/README.md

```diff
@@ -0,0 +1,162 @@
+# Docker Compose Deployment
+
+The CCNP can be deployed in the TDVM by docker compose.
+
+## 1. Prerequisite
+
+### 1.1 Host Side 
+
+The host should enable TDX, and setup service PCCS & service QGS which help generate TDX quote in the guest.
+
+### 1.2 Guest Side
+
+The TDX device in the guest has different names in certain versions
+- `/dev/tdx-attest`: the very early name, the version is not supported. 
+- `/dev/tdx-guest` or `/dev/tdx_guest`: the name for newer version, it is supported.
+
+To enable the containers read and write the TDX device, change the access privilege of the TDX device.
+
+```
+chmod 0666 $(find /dev/ -name "tdx*")
+```
+
+This tool will generate files to create some docker composes which are place in `/tmp/docker_ccnp`. Please make sure the directory clear.
+
+For convenience, run the script `prepare.sh` directly as root.
+
+```
+./prepare.sh
+```
+
+## 2. Use CCNP
+
+### 2.1 Deploy CCNP
+
+Use the script `deploy-ccnp.sh` to deploy the CCNP services. 
+
+```
+./deploy-ccnp.sh
+```
+
+In default, the script will launch three containered services
```

Comment:
think "three container services" will show what you want to say

Reply:
A typo, which should be `three containerized services`